Harvard Caselaw Access Project.js

```diff
@@ -0,0 +1,175 @@
+{
+	"translatorID": "2a1cafb9-6f61-48d3-b621-c3265fde9eba",
+	"label": "Harvard Caselaw Access Project",
+	"creator": "Franklin Pezzuti Dyer",
+	"target": "^https://(.*\\.)*case\\.law",
```

Comment:
is this every anything other then cite.case.law? If not, let's just use `(cite\\.)?` (also, don't use `* `where `?` suffices)

Reply:
It can also be just `case.law`, namely on the search pages. I can make this more specific by narrowing it down to `(cite\\.)?case\\.law`.